bundles/org.openhab.binding.mqtt.espmilighthub/src/main/java/org/openhab/binding/mqtt/espmilighthub/internal/handler/EspMilightHubHandler.java

```diff
@@ -537,6 +537,8 @@ public void bridgeStatusChanged(ThingStatusInfo bridgeStatusInfo) {
                 return;
             }
             this.connection = connection;
+            updateStatus(ThingStatus.UNKNOWN, ThingStatusDetail.NOT_YET_READY,
+                    "Waiting for 'milight/status: connected' MQTT message to be recieved. Check hub has 'MQTT Client Status Topic' configured.");
```

Comment:
Typo: received

Reply:
I believe `ThingStatusDetail.NOT_YET_READY` was introduced by #3330 and used in combination with `ThingStatus.UNINITIALIZED` by the framework:

https://github.com/openhab/openhab-core/blob/cdab53b814c08380bba52a3f5f31543117be52cb/bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/internal/ThingManagerImpl.java#L926-L931

So I'm not sure it's intended the be used by bindings explicitly and in this combination with `ThingStatus.UNKNOWN`. Maybe the documentation needs to be updated: https://next.openhab.org/docs/concepts/things.html#status-details

@J-N-K - perhaps you can comment?

Reply:
I think you're right @jlaur

Reply:
I just grep our binding database and it is clean, NOT_YET_READY is not used.